src/_css/main.css

```diff
@@ -301,6 +301,9 @@ main a:hover code, main a:focus code {
   color: var(--white);
   background-color: var(--darkblue);
 }
+.note pre[class^="language"] {
```

Comment:
Do we need the attribute selector? Wouldn't `.note pre` work?

Reply:
It's just a safety net in case someone includes a non-syntax-highlighted `pre` block. It could probably be removed.

Reply:
That's what I meant: we'd want those non-syntax highlighted `pre` blocks to also render correctly, right?

Reply:
Let me fix it.